docker/test/integration/minifi/processors/PutAzureBlobStorage.py

```diff
@@ -0,0 +1,13 @@
+from ..core.Processor import Processor
+
+
+class PutAzureBlobStorage(Processor):
+    def __init__(self):
+        super(PutAzureBlobStorage, self).__init__('PutAzureBlobStorage',
+                                                  properties={
+                                                      'Container Name': 'test_container',
+                                                      'Connection String': 'DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azure-storage-server:10000/devstoreaccount1;QueueEndpoint=http://azure-storage-server:10001/devstoreaccount1;',
+                                                      'Blob': 'test_blob',
+                                                      'Create Container': 'true',
+                                                  },
+                                                  auto_terminate=['success'])
```

Comment:
Does this processor not have a failure relationship as well?

Reply:
Updated in [48ec7a](https://github.com/apache/nifi-minifi-cpp/pull/979/commits/48ec7a2ebb6d109c147f2a3b83aa2ae8996acfa2)